### ISSUE TITLE
resource.add_role not available when using resourcify

### ISSUE THREAD

**Issue by philjeffs** [RolifyCommunity/rolify#69]
I added resourcify to my User model, and ran:

u = User.first
u.add_role :admin

I get an error, "undefined method `add_role' ".

If remove resourcify from my model and add in rolify, then it works fine, but then I don't have access to User.with_role. 

I'm using Rolify 3.1.0.

**Comment by ewilliam** [comment]
what if I want to do this

``` ruby
class User < ActiveRecord::Base
  resourcify
  rolify
end
```
so I can do things like

```
User.add_role :admin, User
User.add_role :vendor, Plan.first
User.find_roles
```

how do I do this? in the code as is, ```User.find_roles``` returns a NoMethodError.

**Comment by beverlyguillermo** [comment]
@ewilliam A fix for your problem (which was the same as mine) was merged into master -- #343

**Comment by virtual-monk** [comment]
@ewilliam Did the merge to master resolve your issue? I'm trying to do the same thing, with little luck
